docs/triggers.md

```diff
@@ -59,6 +59,7 @@ Trigger on a Dapr topic subscription.  The trigger will automatically expose and
 ```json
 {
     "type": "daprTopicTrigger",
+    "pubsubname": "pubsub",
```

Comment:
Do we have a typical name we use for pubsubs? ex: does dapr set you up with one called something by default?

Reply:
Yes, the cli default generated via `dapr init` is simply named `pubsub`.

Reply:
Ok thanks that's what I remembered.